apps/web/src/hooks/useFindSessionQuery.ts

```diff
@@ -36,10 +23,14 @@ export const useFindSessionQuery = (
   return useQuery({
     enabled,
     queryFn: async () => {
-      const response = await axios.get('/v1/sessions/', {
+      const response = await axios.get('/v1/sessions', {
         params
       });
-      return $SessionWithUser.array().parseAsync(response.data);
+      const parsedData = $SessionWithUser.array().safeParseAsync(response.data);
```

Comment:
how is this an improvement compared to the Zod error? This error message is terrible.